### ISSUE TITLE
Hair colliding with breasts though the back

### ISSUE THREAD

**Issue by billyboy1122** [ManlyMarco/KKABMX#11]
I've found an issue with their hair collision when it comes to larger breasts. When "breast scale 1" is used to enlarge the breasts beyond stock, the back hair starts colliding with the breasts through the character's back! Front hair seems to be ok. Using "breast collision scale" doesn't change anything. Seems like one of the colliders is too big perhaps?

**Comment by ManlyMarco** [comment]
It might be because the collision slider is only restricted in one direction. Can you check if this can be fixed by turning on the advanced bonemod window and tweaking the cf_hit_bust02_L / R XYZ sliders there?

**Comment by billyboy1122** [comment]
Sorry, I should have clarified I'm talking about AI Shoujyo here, not Koikatsu. I've got both "Show Advanced Bonemod Window" and "Use XYZ scale sliders" set to "true" in the config, but I only see the XYZ sliders, no advanced window. Either way, the lowest I can set "breast collision scale" is "-10" and this doesn't solve the issue. I've also tried using SBU3 to change the scale of "cf_hit_Mune02_s_L / R" to as low as "0.001" but this doesn't change anything.

**Comment by ManlyMarco** [comment]
Toggle the `Advanced Bonemod Window` setting a couple times to make it show up, it's a known bug that's not fixed because the setting is temporary until the full game drops.

In this case it might be some other collider that is causing this. The colliders are set up differently from koi so this particular collider might be a red herring.

**Comment by billyboy1122** [comment]
Maybe I'm missing something, but what toggle? I can't find anything in the game to show the advanced bonemod window. Did you mean alternating setting "Show Advanced Bonemod Window" in "KKABMX.Core" to "true" and "false" when you start the game? I tried that.

**Comment by ManlyMarco** [comment]
https://streamable.com/5c932

**Comment by billyboy1122** [comment]
I give up, how are you getting that window to pop up? I've read all the readmes I can find and mashed every key on my keyboard, but the best I can get is the config window the autotranslator to appear! 

Also, I had took a look at the bone that it appears KKABMX uses for "Breast Collision" (cf_hit_Mune02_s_L \ R) and noticed that the Y scale is set to 1 instead of 0.6 like X and Z. Changing it to 0.6 didn't fix anything though.

**Comment by ManlyMarco** [comment]
Try updating your bepinex and plugins, there was an update to BR not long ago so you can get that.

**Comment by billyboy1122** [comment]
I've tried updating Bepinex, but that hasn't caused the menu to appear. I did try a fresh install, but that gives me a "Failed to find a BoneController or there are no bone modifiers" error. I had to copy across the new version of bepinex and replace the old one to make the mod load.

Here's an example card, the hair will suddenly fly away when the breasts are at different positions.
![AISChaF_20191022190829737](https://user-images.githubusercontent.com/52345884/67276522-d3dfad00-f4ff-11e9-9950-ff74fc39249f.png)
